lib/handleRequest.js

```diff
@@ -57,16 +57,20 @@ function handleRequest (err, request, reply) {
 }
 
 function handler (request, reply) {
-  if (reply.context.preValidation !== null) {
-    hookRunner(
-      reply.context.preValidation,
-      hookIterator,
-      request,
-      reply,
-      preValidationCallback
-    )
-  } else {
-    preValidationCallback(null, request, reply)
+  try {
+    if (reply.context.preValidation !== null) {
+      hookRunner(
+        reply.context.preValidation,
+        hookIterator,
+        request,
+        reply,
+        preValidationCallback
+      )
+    } else {
+      preValidationCallback(null, request, reply)
+    }
+  } catch (err) {
+    reply.code(500).send(err)
```

Comment:
```suggestion
    preValidationCallback(err, request, reply)
```

In this way, there are more checks before running the send